src/coreclr/tools/aot/ILCompiler/ILCompiler.csproj

```diff
@@ -10,8 +10,8 @@
   <!-- BEGIN: Workaround for https://github.com/dotnet/runtime/issues/67742 -->
   <PropertyGroup Condition="'$(BuildingInsideVisualStudio)' != 'true'">
     <PublishDir>$(RuntimeBinDir)ilc-published/</PublishDir>
-    <NativeAotSupported Condition="'$(TargetOS)' != 'windows' and '$(TargetOS)' != 'linux' and '$(TargetOS)' != 'osx'">false</NativeAotSupported>
-    <NativeAotSupported Condition="'$(TargetArchitecture)' != 'x64'">false</NativeAotSupported>
+    <!-- Can't use NativeAOT in source build yet https://github.com/dotnet/runtime/issues/66859 -->
```

Comment:
I made this to match https://github.com/dotnet/runtime/blob/161b12c287b5d402c60a03314f597d27ad752754/src/coreclr/tools/aot/crossgen2/crossgen2.csproj#L7-L8